test/mpi/test_case.cc

```diff
@@ -40,8 +42,13 @@ std::shared_ptr<TestCase> TestCase::init(ucc_coll_type_t _type,
         return std::make_shared<TestReduce>(msgsize, inplace, dt, op, mt, root,
                                            _team, max_size);
     case UCC_COLL_TYPE_ALLTOALL:
-        return std::make_shared<TestAlltoall>(msgsize, inplace, mt, _team,
-                                              max_size);
+        if (is_onesided) {
+            return std::make_shared<TestAlltoall>(msgsize, inplace, mt, _team,
+                                                  max_size, onesided_buffers);
+        } else {
+            return std::make_shared<TestAlltoall>(msgsize, inplace, mt, _team,
```

Comment:
need some special string for this test in TestCase::str()

Reply:
done